### ISSUE TITLE
News downloader crashes when there is no koreader/news/feed_config.lua

### ISSUE THREAD

**Issue by mwoz123** [koreader/koreader#6002]
* KOReader version: last nighty 
* Device: android 

#### Issue
 Discovered in #6001. 

When news downloader config file(feed_config.lua) was manually moved /removed from koreader/news dir and there current news download folder doesn't exist, news downloader crashes and doesn't show up in plugins menu. 

To do: before copying it could be checked if file exists. 

After I did into that and described all of that I'm wondering if it's worth to handle it..
@Frenzie?

**Comment by Frenzie** [comment]
I don't think I fully understand the steps. If I select a custom directory, close KOReader and then remove it the directory, that whole directory tree is recreated automatically.

**Comment by mwoz123** [comment]
Yes but if your move/remove it from /sdcard/koreader/news/feed_config.lua
(is it /sdcard/,  /storage/, or /somewhere else where root koreader dir is on emulated SD?) as you have access to it and can change it, it will fail.
There can be extra scenarios but at least similar to this. 

On the other hand anything you remove from your koreader root dir will probably cause koreader to fail. So I'm not sure..

**Comment by Frenzie** [comment]
Any settings file can be safely removed to reset it. A crash is a bug.
